src/main/java/cz/muni/fi/pv286/mmt/model/Messages.java

```diff
@@ -0,0 +1,19 @@
+package cz.muni.fi.pv286.mmt.model;
+
+/**
+ * This class contains output messages used in application.
+ */
+public final class Messages {
+    public static final String MESSAGE_INVALID_ARGUMENTS = "Invalid arguments";
+    public static final String FILE_IS_DIRECTORY = "File is a directory";
+    public static final String FILE_SET_DUPLICITY = "File set duplicity";
+    public static final String CANT_READ_FILE = "Cannot read file";
+    public static final String CANT_WRITE_FILE = "Cannot write file";
+    public static final String FORMAT_SET_DUPLICITY = "Format set duplicity";
+    public static final String UNKNOWN_OPTION = "Unknown option provided";
+    public static final String UNKNOWN_FORMAT = "Unknown format provided";
+    public static final String UNKNOWN_ARGUMENT = "Unknown argument provided";
```

Comment:
**praise:** I like that you created class containing constants for error messages